app/views/articles/_single_story.html.erb

```diff
@@ -65,7 +65,15 @@
              tags_to_display = tags_to_display.reject { |tag| tag == flare_tag.name }
            end %>
         <% tags_to_display.each do |tag| %>
-        <a href="/t/<%= tag %>" class="crayons-tag"><span class="crayons-tag__prefix">#</span><%= tag %></a>
+          <% if tag === @tag_model.name %>
```

Comment:
I wonder if we really need that triple equals condition here, I mean, It would make sense if the `tag` was a regex and `@tag_mode.name` was a text. I don't think that's the case here.

```ruby
/testing/ === 'Hey, I am just testing...' => true
```